docroot/profiles/custom/cgov_site/modules/custom/cgov_core/cgov_core.module

```diff
@@ -848,3 +851,50 @@ function cgov_core_taxonomy_term_delete(EntityInterface $entity) {
     }
   }
 }
+
+/**
+ * Implements hook_ENTITY_TYPE_insert().
+ */
+function cgov_core_redirect_insert(Redirect $entity) {
+  $moduleHandler = \Drupal::service('module_handler');
+  if (!$moduleHandler->moduleExists('purge')) {
+    return;
+  }
+    // The value for cgdp.cache_url_host is computed in cgov_core.settings.php.
+    $host = Settings::get('cgdp.cache_url_host');
+    if (empty($host)) {
+      $host = \Drupal::request()->getSchemeAndHttpHost();
+    }
```

Comment:
Please remove this if block and set `cgdp.cache_url_host` to the appropriate value in `cgov_core.settings.php`.  (See my other comment.)